docs/gds-supported-platform/bootstrapping-clusters.md

```diff
@@ -6,6 +6,18 @@
   * Set `config-version` to the branch name you chose.
   * Set `config-path` to the path to your new cluster YAML file, relative to the root of tech-ops-cluster-config.
   * Set `config-values-path` to the path of your new cluster values YAML file, relative to the root of tech-ops-cluster-config.
+  * Set the following vars:
+      * account-id: "value from PR #268 example"
```

Comment:
just say 011571571136, it's in the gsp.git examples

Reply:
agreed and edited to suit.